Pull

SherfeyInv opened the pull request.

imgbot[bot] committed:
[ImgBot] Optimize images (#1)

*Total -- 32.05kb -> 12.49kb (61.03%)

/build_assets/images/toolbarLogClear-orig.gif -- 0.90kb -> 0.16kb (81.96%)
/build_assets/images/dmg_background.png -- 11.52kb -> 2.42kb (79.01%)
/build_assets/images/arelle.gif -- 1.42kb -> 0.32kb (77.37%)
/build_assets/images/xbrl.gif -- 0.91kb -> 0.22kb (75.73%)
/build_assets/images/xbrl128.gif -- 0.91kb -> 0.22kb (75.73%)
/build_assets/images/toolbarFindMenu.gif -- 0.93kb -> 0.34kb (63.2%)
/build_assets/images/toolbarCompare.gif -- 0.95kb -> 0.36kb (61.82%)
/build_assets/images/toolbarValidate.gif -- 0.96kb -> 0.37kb (61.36%)
/build_assets/images/toolbarLogClear.gif -- 0.99kb -> 0.39kb (60.08%)
/build_assets/images/xbrl128-2.gif -- 1.56kb -> 0.73kb (53.07%)
/build_assets/images/arelle32.gif -- 1.18kb -> 0.57kb (51.37%)
/build_assets/images/octocat.png -- 9.39kb -> 6.01kb (35.96%)
/build_assets/images/toolbarDelete.gif -- 0.19kb -> 0.14kb (26.67%)
/build_assets/images/columnSortDown.gif -- 0.06kb -> 0.05kb (10%)
/build_assets/images/columnSortUp.gif -- 0.06kb -> 0.05kb (10%)
/build_assets/images/toolbarProperties.gif -- 0.12kb -> 0.11kb (9.45%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

SherfeyInv committed:
Create python-publish.yml

SherfeyInv committed:
Create python-package-conda.yml

SherfeyInv committed:
Create aws.yml

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

SherfeyInv committed:
Modified by www.SourceFiles.app

aviary2-wf commented:
## Security Insights
<details>
<summary>Error scanning for security relevant changes</summary>
</details>
<details>
<summary>Error scanning for image validation issues</summary>
</details>
<details>
<summary>Error scanning for cfn validation issues</summary>
</details>
<details>
<summary>Error scanning for security relevant context</summary>
</details>

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

SherfeyInv committed:
Create django.yml

SherfeyInv reviewed (commented):
Added AWS workflows information

SherfeyInv committed:
Create dependabot.yml

Commit:
Sherfey_SAM_FPS Responsibility and Integrity Records All Data 2024-09-18 09.30.01 copy.xlsx

SherfeyInv committed:
mainMarketNotices.rss

SherfeyInv committed:
Merge branch 'master' of git@github.com:SherfeyInv/ArelleBuilder

SherfeyInv committed:
Sherfey-d89c41b2-4c4e-42ba-ba40-86cfca6c3700 copy.xlsx

SherfeyInv committed:
SAM_FPS Responsibility and Integrity Records All Data 2024-12-16 09.30.03.xlsx

SherfeyInv committed:
AysherIntelSearch.xlsx

SherfeyInv committed:
aysherSearch 2.xlsx

SherfeyInv committed:
IssuerTemplateFor288222 copy.xlsx

SherfeyInv committed:
employed-and-office-of-e.svg

SherfeyInv committed:
spdji-methodology-and-regulatory-status-report-european-union copy.xlsx

SherfeyInv committed:
IssuerTemplateFor288222RS.xlsx

SherfeyInv committed:
tmus-geo-ip.txt

SherfeyInv committed:
pki101.md

SherfeyInv committed:
TablesEU

SherfeyInv committed:
data.pdf

SherfeyInv committed:
Data Release.xlsx

SherfeyInv committed:
historical_indicative_sofr_avg_ind_data.xlsx

SherfeyInv committed:
IssuerTemplateFor288631_Aysher.xlsx

SherfeyInv committed:
CCL_Public_Dashboard_Agg_3 copy.xlsx

SherfeyInv committed:
Merge branch origin/SherfeyInv-patch-1 into master

SherfeyInv committed:
all-important-notices.webarchive

snyk-bot committed:
fix: requirements_mac.txt to reduce vulnerabilities


The following vulnerabilities are fixed by pinning transitive dependencies:
- https://snyk.io/vuln/SNYK-PYTHON-PYMYSQL-6968882
